pages/api/auth/logout.ts

```diff
@@ -0,0 +1,23 @@
+import { createDeleteSessionCookie, SESSION_TOKEN_COOKIE } from '@lib/helpers/backend/session-cookie';
+import { HttpMethod } from '@lib/http-method';
+import { deleteSession } from '@lib/services/session';
+import { StatusCodes } from 'http-status-codes';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  switch (req.method as HttpMethod) {
+    case HttpMethod.POST:
+      return logout(req, res);
+    default:
+      return res.status(StatusCodes.INTERNAL_SERVER_ERROR).end();
```

Comment:
Same comment as for other endpoint; does 400 make sense here? There is probably a convention I'm unaware of.